src/python/dxpy/templating/templates/nextflow/dxapp.json

```diff
@@ -42,12 +42,26 @@
       "help": "Shows additional information in the job logs. If true, the execution log messages from Nextflow will also be included.",
       "class": "boolean",
       "default": false
+    },
+    {
+      "name": "resume",
+      "label": "Resume",
+      "help": "Unique ID of the previous session to be resumed. If not provided, will resume the last session run by an applet with the same name in the current project that runs during the last 6 months",
```

Comment:
According to the spec the wording should be a little different:
```Unique ID of the previous session to be resumed. If 'true' or 'last' is provided instead of the sessionID, will resume the latest resumable session run by an applet with the same name in the current project in the last 6 months.```